### ISSUE TITLE
Problem with maximize

### ISSUE THREAD

**Issue by newptcai** [JuliaIntervals/IntervalOptimisation.jl#46]
When I tried, I got
```
julia> maximise(X->((x,y)=X; x/(y+1)+y/(x+1)), IntervalBox(0..1, 2), tol=1e-5)
ERROR: MethodError: no method matching minimise(::IntervalOptimisation.var"#5#6"{var"#31#32"}, ::IntervalBox{2,Float64}, ::Type{HeapedVector}, ::Float64)
Closest candidates are:
  minimise(::Any, ::T; structure, tol) where T at /home/xinca341/.julia/packages/IntervalOptimisation/A3NrJ/src/optimise.jl:19
Stacktrace:
 [1] #maximise#4(::Type, ::Float64, ::typeof(maximise), ::var"#31#32", ::IntervalBox{2,Float64}) at /home/xinca341/.julia/packages/IntervalOptimisation/A3NrJ/src/optimise.jl:61
 [2] (::IntervalOptimisation.var"#kw##maximise")(::NamedTuple{(:tol,),Tuple{Float64}}, ::typeof(maximise), ::Function, ::IntervalBox{2,Float64}) at ./none:0
 [3] top-level scope at REPL[21]:1
```

I am using `IntervalOptimisation v0.4.0`

**Comment by dpsanders** [comment]
That works for me with version 0.4.1.

**Comment by dpsanders** [comment]
```jl
julia> maximise(X->((x,y)=X; x/(y+1)+y/(x+1)), IntervalBox(0..1, 2), tol=1e-5)
([0.999997, 1.00001], IntervalBox{2,Float64}[[0.999991, 1] × [0.999991, 1], [0.999991, 1] × [0, 6.67704e-06], [0, 6.67704e-06] × [0.999991, 1], [0.999982, 0.999992] × [0.999991, 1], [0.999991, 1] × [0.999982, 0.999992], [0.999991, 1] × [6.67703e-06, 1.34593e-05], [6.67703e-06, 1.34593e-05] × [0.999991, 1], [0.999991, 1] × [0.999974, 0.999983], [0.999974, 0.999983] × [0.999991, 1], [0.999982, 0.999992] × [0.999982, 0.999992], [0.999965, 0.999975] × [0.999991, 1], [0.999991, 1] × [0.999965, 0.999975]])
```

**Comment by dpsanders** [comment]
Oops, 0.4.1 hasn't been released yet... Sorry!

Try updating once [this](https://github.com/JuliaRegistries/General/pull/6726) has been merged (or just use master).

**Comment by newptcai** [comment]
Thanks!

**Comment by dpsanders** [comment]
Fixed by 0.4.1.
